### ISSUE TITLE
Schema prefix support for Ecto

### ISSUE THREAD

**Issue by dlobo** [tompave/fun_with_flags#86]
Would it be possible to add schema support for ecto via prefixes so funwithflags can live in a separate schema in postgres. Useful in a multi-tenancy situation.

I can work around it by customizing the persistence adapter, but it is a fairly simple fix, and happy to submit a patch (I don't know about how it would work in MySQL). Will require changes in two files: ecto.ex and config.ex and also introduce a new name called ecto_table_prefix

lobo

**Comment by tompave** [comment]
Hello, thank you for opening the issue.

This was already discussed in https://github.com/tompave/fun_with_flags/issues/83. My recommendation (first reply) was to do something similar to what you suggest: introduce a custom persistence adapter (you can fork the current Ecto adapter) and then configure the package to use it.

**Comment by dlobo** [comment]
thanx for the response and no worries. I went ahead with the custom persistence adapter and it works quite nicely. Here's the code in case someone is looking for the changes:

https://github.com/glific/glific/blob/feature/cron/lib/glific/flags/ecto.ex

(note that you'll need to replace feature/cron in a couple of days with master, once the branch has merged)

FYI, I also got some dialyzer (and credo) errors on that file. I cleaned up the credo errors, but skipped the dialyzer ones

thanx for the library
